app/constants/chartTypeConfigs/discreteBarChart.js

```diff
@@ -14,7 +14,11 @@ export const defaultOpts = {
   annotations: {
     selector: '.nv-bar rect',
     container: '.nv-barsWrap',
-    getCoords: function getCoords(el) {
+    getCoords: function getCoords({ data }, selector) {
```

Comment:
What is the first argument to this function? If you destructure in the function body it would be more clear what is passed into `getCoords`.